cmd/gardener-extension-shoot-lakom-service/app/app.go

```diff
@@ -116,6 +118,12 @@ func (o *Options) run(ctx context.Context) error {
 	if err := monitoringv1.AddToScheme(mgr.GetScheme()); err != nil {
 		return fmt.Errorf("could not update manager scheme: %w", err)
 	}
+	if err := v1alpha1apilakom.AddToScheme(mgr.GetScheme()); err != nil {
+		return fmt.Errorf("could not update manager scheme: %w", err)
+	}
+	if err := apilakom.AddToScheme(mgr.GetScheme()); err != nil {
+		return fmt.Errorf("could not update manager scheme: %w", err)
```

Comment:
Update the error messages so that they distinguish when different APIs are added to the scheme.